src/main/java/net/dries007/tfc/world/biome/BiomeNoise.java

```diff
@@ -170,6 +170,45 @@ public static Noise2D lowlands(long seed)
             );
     }
 
+    /**
+     * Very flat biome
+     */
+    public static Noise2D flats(long seed)
+    {
+        return new OpenSimplex2D(seed)
+                .octaves(4)
+                .spread(0.03f)
+                .scaled(SEA_LEVEL_Y - 12, SEA_LEVEL_Y + 8)
+                .clamped(SEA_LEVEL_Y, SEA_LEVEL_Y + 2);
```

Comment:
Formatting is inconsistent with other lines
```suggestion
        return new OpenSimplex2D(seed)
            .octaves(4)
            .spread(0.03f)
            .scaled(SEA_LEVEL_Y - 12, SEA_LEVEL_Y + 8)
            .clamped(SEA_LEVEL_Y, SEA_LEVEL_Y + 2);
```